src/Agent.Worker/CodeCoverage/CodeCoverageUtilities.cs

```diff
@@ -110,6 +107,193 @@ public static int GetPriorityOrder(string coverageUnit)
             return (int)Priority.Other;
         }
 
+        public static string ThrowIfParameterEmpty(string parameterValue, string parameterName)
+        {
```

Comment:
or "TrimNonEmptyParam". If it's being called with Empty param, it's meant to throw error